samples/KvsAudioVideoStreamingSample.c

```diff
@@ -189,7 +191,7 @@ INT32 main(INT32 argc, CHAR* argv[])
     STRNCPY(audioCodec, AUDIO_CODEC_NAME_AAC, STRLEN(AUDIO_CODEC_NAME_AAC)); //aac audio by default
 
     if (argc == 6) {
-        if (STRCMP(argv[5], "1")){
+        if (!STRCMP(argv[5], "1")){
```

Comment:
Can we update this to a string like "enable_events" or something? 
"1" seems to be unclear...

Also, why do we want gEventsEnabled to be UINT8 and not BOOL?

We should also update the README to include details on this option.

Reply:
Will update